src/bufr/BufrParser/Exports/Variables/DatetimeVariable.cpp

```diff
@@ -87,17 +87,19 @@ namespace Ingester
         std::vector<int64_t> timeOffsets;
         timeOffsets.reserve(map.at(getExportKey(ConfKeys::Year))->size());
 
+        auto yearVar = map.at(getExportKey(ConfKeys::Year));
+
         // Validation
-        if (map.at(getExportKey(ConfKeys::Year))->getDims().size() != 1 ||
-            map.at(getExportKey(ConfKeys::Month))->getDims().size() != 1 ||
-            map.at(getExportKey(ConfKeys::Day))->getDims().size() != 1 ||
+        if (!yearVar->hasSamePath(map.at(getExportKey(ConfKeys::Year))) ||
```

Comment:
Don't you want yearVar to haveSamePath as ConfKeys year in this case? The ! in the front would say that you don't. If not, why's that?

Reply:
This was a little silly as it would always have the same path. This iff statement looks for an inconsistency in file paths an creates an exception if there is one.